### ISSUE TITLE
📎 SOW: Accessibility

### ISSUE THREAD

**Issue by jillpe** [notch8/utk-hyku#393]
# Summary
UTK would like their site to be WCAG 2.0 aa compliant and for videos to have closed captioning so their site can be accessible to more people and those with disabilities. 

The accessibility report needs to be reviewed for what items need to be resolved. This section will most likely focus on end front end changes. Accessibility improvements of the viewers/media players will come in later tickets.

Accessibility Audit: 
[Report.for.https___test.hykucommons.org_.pdf](https://github.com/scientist-softserv/utk-hyku/files/11287664/Report.for.https___test.hykucommons.org_.pdf)

# Accepted Criteria
 
~~- [ ] As a user, when I view a video, I have an option to turn on closed captioning and view them along with the video~~
~~- [ ] As a user, when I view a video, I can select the closed captioning language to view~~
- [ ] Issues found, based on the results of a WCAG 2.0 aa audit, are resolved 

# To Do
- #41
- #43
- #442 
- #387 

# Testing Instructions

# Notes
- May require more work to make compatible with AllinsonFlex
- [Audit](https://gitlab.com/notch8/utk-hyku/uploads/43ce69f5a8cec6f10387dd34c3b15594/Report_for_https___test.hykucommons.org_.pdf)

**Comment by markpbaggett** [comment]
@jillpe this looks good. Just adding that videos may have captions in multiple languages and the user should be able to set the one that they want.

**Comment by jillpe** [comment]
confirmed client meeting 2023-06-21, we can do the same thing we are doing with the audio transcripts: we just need them to be available in the manifest, so when the UV is capable of doing closed captions it can pull from that. We do not need to get closed captions to appear in the UV as part of this work. We can remove the first 2 criteria in the acceptance criteria:

- As a user, when I view a video, I have an option to turn on closed captioning and view them along with the video
- As a user, when I view a video, I can select the closed captioning language to view
